code/modules/vore/eating/belly_obj_vr.dm

```diff
@@ -350,7 +349,7 @@
 			for(var/slot in slots)
 				var/obj/item/thingy = M.get_equipped_item(slot = slot)
 				if(thingy)
-					M.unEquip(thingy,force = TRUE)
+					M.u_equip(thingy)
```

Comment:
The real fix for this is making unEquip use drop_location(), this is sort of a hack I'm not sure if I like.